css/components/image.css

```diff
@@ -0,0 +1,29 @@
+.img-sm{
+    width: 200px;
+    height: 200px;
+}
+
+.img-md{
+    width: 250px;
+    height: 250px;
+}
+
+.img-lg{
+    width: 300px;
+    height: 300px;
+}
+
+.resp-img{
+    max-inline-size: 100%;
+    width: 100%;
```

Comment:
To make your image responsive you can use max-width that works well

Reply:
Thanks. looking into it.